src/index.html

```diff
@@ -8,5 +8,69 @@
   </head>
   <body>
     <h1>Calendar</h1>
+
+    <main class="calendar">
+      <div class="calendar__day calendar__day--number--1 calendar__day--week--mon"></div>
+      
+      <div class="calendar__day calendar__day--number--2 calendar__day--week--tue"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
don't set modifiers for each day. only the calendar should have modifiers

Reply:
the html should looks like here
![image](https://user-images.githubusercontent.com/98764641/214865723-ec326481-950f-40c5-9eb8-47e70b23d840.png)